mantidproject/mantid
# Add support for the ILL on IDR

Fixes [#11162](http://trac.mantidproject.org/mantid/ticket/11162).

Note this still just supports IN16B (for now).

To test:
- Set facility to ILL, open IDR
- Load `ILLIN16B_034745.nxs`
- Enable Mirror Mode
- Enable Plot Result
- Run (should get  plot of an energy transfer)
